aries-site/src/data/structures/components.js

```diff
@@ -717,6 +717,7 @@ export const components = [
   },
   {
     name: 'CheckBoxGroup',
+    accessibility: 'Failed WCAG 2.2 A',
```

Comment:
```suggestion
    accessibility: 'Passed WCAG 2.2 AA',
```
If you agree with my suggestion to change 1.3.1 to passed then we should update this to passed